Image processing/Image compression analyser/visualizer/diff_generator.py

```
@@ -0,0 +1,263 @@
+        # Ensure both images are the same size
+        width, height = original.size
+        compressed = compressed.resize((width, height))
+        
+        # Create a new image with double width
+        comparison = Image.new('RGB', (width * 2, height + 50))
+        draw = ImageDraw.Draw(comparison)
+        
+        # Add title
+        try:
+            font = ImageFont.truetype("arial.ttf", self.font_size)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)